Modules/Core/SpatialObjects/include/itkEllipseSpatialObject.h

```diff
@@ -100,13 +100,25 @@ class ITK_TEMPLATE_EXPORT EllipseSpatialObject : public SpatialObject<TDimension
   using Superclass::IsInsideInObjectSpace;
 
 #if !defined(ITK_LEGACY_REMOVE)
-  itkLegacyMacro(void SetRadius(double radius)) { this->SetRadiusInObjectSpace(radius); }
-
-  itkLegacyMacro(void SetRadius(ArrayType radii)) { this->SetRadiusInObjectSpace(radii); }
-
-  itkLegacyMacro(ArrayType GetRadius() const) { return this->GetRadiusInObjectSpace(); }
-
-  itkLegacyMacro(void SetRadiiInObjectSpace(ArrayType radii)) { this->SetRadiusInObjectSpace(radii); }
+  itkLegacyMacro(void SetRadius(double radius))
```

Comment:
This set of changes seems unrelated.

Reply:
Thanks! Honestly I have no idea why clang-format decided to make those unrelated changes  🤷

Reply:
- @dzenanz Related pull request, doing those unrelated changes: #4857

Reply:
- Resolved by pull request #4857